src/e2e-test/features/gcs/source/GCSToGCSAdditonalTests.feature

```diff
@@ -0,0 +1,116 @@
+@GCS_Source
+Feature: GCS source - Verification of GCS to GCS Additional Tests successful
+
+  @GCS_CSV @GCS_SINK_TEST @EXISTING_GCS_CONNECTION
+  Scenario: To verify data is getting transferred from GCS to GCS using test Schema Detection On Single File with connection
+    Given Open Datafusion Project to configure pipeline
+    When Select plugin: "GCS" from the plugins list as: "Source"
+    When Expand Plugin group in the LHS plugins list: "Sink"
+    When Select plugin: "GCS" from the plugins list as: "Sink"
+    Then Connect plugins: "GCS" and "GCS2" to establish connection
+    Then Navigate to the properties page of plugin: "GCS"
+    Then Select dropdown plugin property: "select-schema-actions-dropdown" with option value: "clear"
+    Then Click plugin property: "switch-useConnection"
+    Then Click on the Browse Connections button
+    Then Select connection: "gcsConnectionName"
+    Then Enter input plugin property: "referenceName" with value: "sourceRef"
+    Then Enter GCS source property path "gcsCsvDataFile"
+    Then Select GCS property format "csv"
+    Then Validate "GCS" plugin properties
+    Then Verify the Output Schema matches the Expected Schema: "gcsSingleFileDataSchema"
+    Then Close the Plugin Properties page
+    Then Navigate to the properties page of plugin: "GCS2"
+    Then Click plugin property: "switch-useConnection"
+    Then Click on the Browse Connections button
+    Then Select connection: "gcsConnectionName"
+    Then Enter input plugin property: "referenceName" with value: "sourceRef"
+    Then Enter GCS sink property path
+    Then Select dropdown plugin property: "select-format" with option value: "csv"
+    Then Validate "GCS" plugin properties
+    Then Close the Plugin Properties page
+    Then Save the pipeline
+    Then Preview and run the pipeline
+    Then Wait till pipeline preview is in running state
+    Then Open and capture pipeline preview logs
+    Then Verify the preview run status of pipeline in the logs is "succeeded"
+    Then Close the pipeline logs
+    Then Click on preview data for GCS sink
+    Then Verify preview output schema matches the outputSchema captured in properties
+    Then Close the preview data
+    Then Deploy the pipeline
+    Then Run the Pipeline in Runtime
+    Then Wait till pipeline is in running state
+    Then Open and capture logs
+    Then Verify the pipeline status is "Succeeded"
+    Then Validate the data from GCS Source to GCS Sink with expected csv file and target GCS bucket
+
+  @GCS_CSV @GCS_SINK_TEST
+  Scenario: To verify data is getting transferred from GCS to GCS using test Schema Detection On Single File
```

Comment:
Add without connection in scenario as it is not using connection

Reply:
Added without connection in the scenarios for more visibility.